test/testutil.py

```diff
@@ -75,3 +80,27 @@ def can_start_rootful_containers():
             return res.stdout.strip() == "true"
         case unknown:
             raise ValueError(f"unknown platform {unknown}")
+
+
+def write_aws_creds(path):
+    key_id = os.environ.get("AWS_ACCESS_KEY_ID")
+    secret_key = os.environ.get("AWS_SECRET_ACCESS_KEY")
+    if not key_id or not secret_key:
+        print("WARNING: aws credentials not available")
```

Comment:
Should this be an error (i.e. raise here)?

Reply:
I was thinking whether this should be an error or not and thinking about it again (and seeing how it actually failed because the secrets aren't set), I can't think of a good reason for it being just a warning.

Reply:
Done.